elyra/pipeline/pipeline_definition.py

```diff
@@ -317,8 +319,12 @@ def __init__(
             # supporting passing the pipeline definition directly
             self._pipeline_definition = pipeline_definition
 
+        validation_issues = []
         if validate:
-            self.validate()
+            validation_issues = self.validate()
+
+        if not validation_issues and self.get_global_properties():
```

Comment:
Calling `get_global_properties()` seems a little redundant since that's the first thing `propagate_global_properties()` does anyway.
```suggestion
        if not validation_issues:
```

Since `get_global_properties()` is one line, we should consider its removal unless there's a plan to extend it beyond its current implementation.

Reply:
Although not quite the same, [this comment](https://github.com/elyra-ai/elyra/pull/2656#issuecomment-1097141063) by @ptitzler makes a good argument for converting this into a helper method that takes the property name.